docs/discovery/xapi-problem-check.json

```diff
@@ -0,0 +1,107 @@
+{
+	"_comment": "This xAPI statement attempts to fully map all the data represented in the canonical sample event here https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/sample_events.html",
+	"id": "8f87ccde-bb56-4c2e-ab83-44982ef22df0",
+	"actor": {
+		"objectType": "Agent",
+		"account": {
+			"homepage": "https://openedx.org/",
+			"name": "b5059046-d6db-11ec-8490-381428857bd9"
+		}
+	},
+	"verb": {
+		"_comment": "The tracking event type this sample corresponds to is 'problem_check'",
+		"id": "http://adlnet.gov/expapi/verbs/answered",
+		"display": {
+			"en-US": "answered"
+		}
+	},
+	"object": {
+		"objectType": "Activity",
+		"_comment": "This URL is a placeholder utilizing the 'usage_key'.  Should be instance specific, but perhaps transferable across domains?",
+		"id": "https://openedx.org/block-v1:edX+DemoX+Demo_Course+type@problem+block@29c5cbd840324d94be8ba51db1864277",
```

Comment:
If we're using a URL here, it might be useful to prefix `/xblock/` to the path. That has the advantage of namespacing, as well as mapping to a real URL that renders the XBlock in question, e.g. https://demo.openedx.overhang.io/xblock/block-v1:edX+DemoX+Demo_Course+type@problem+block@303034da25524878a2e66fb57c91cf85

![Screen Shot 2022-06-14 at 5 36 17 PM](https://user-images.githubusercontent.com/41625/173693341-8ca83686-28ca-4e57-8d60-421a802d9a32.png)